packages/core/src/locales/ar.ts

```diff
@@ -39,10 +39,10 @@ export const parsedType = (data: any): string => {
 const Nouns: {
   [k in $ZodStringFormats | (string & {})]?: string;
 } = {
-  regex: "input",
-  email: "email address",
-  url: "URL",
-  emoji: "emoji",
+  regex: "نص نمطي",
```

Comment:
I think any translation for "input" would be better here. Just like in the English locale, it would be better in the error message. Also, "نص نمطي" is completely unfamiliar. "مدخل" would be a good alternative.
```suggestion
  regex: "مدخل",
```